ISSUES.md

```diff
@@ -0,0 +1,11 @@
+If you open a GitHub Issue, here is our policy:
+
+    1.  It must be a bug/performance issue or a feature request or a build issue or
+        a documentation issue (for small doc fixes please send a PR instead).
+    2.  Make sure the Issue Template is filled out.
+    3.  The issue should be related to the repo it is created in.
+
+**Here's why we have this policy:** We want to focus on the work that benefits
+the whole community, e.g., fixing bugs and adding features. Individual support
+should be sought on Stack Overflow or other non-GitHub channels. It helps us to
```

Comment:
Maybe here would be a good place to link our other support channels (Discord for example)

StackOverflow is probably not a good source of support for a small project like ours